cspell.json

```diff
@@ -585,6 +585,7 @@
         "unhighlight",
         "unicast",
         "unindent",
+        "Uninstallation",
```

Comment:
Use "uninstalling" instead. This is not a word. If you're in doubt about non-technical words check for them in the [American Heritage Dictionary](https://www.ahdictionary.com/word/) or the [Collins Dictionary](https://www.collinsdictionary.com/us/)

Reply:
Thanks for changing the docs file! Could you also remove it from cspell?

Reply:
Done